wavefront_python_sdk/common/utils.py

```diff
@@ -251,12 +251,16 @@ def tracing_span_to_line_data(name, start_millis, duration_millis, source,
         for uuid in follows_from:
             str_builder.append("followsFrom=" + str(uuid))
     if tags is not None:
+        tag_set = set()
         for key, val in tags:
             if is_blank(key):
                 raise ValueError("Span tag key cannot be blank")
             if is_blank(val):
                 raise ValueError("Span tag val cannot be blank")
-            str_builder.append(sanitize(key) + "=" + sanitize(val))
+            cur_tag = sanitize(key) + "=" + sanitize(val)
+            if cur_tag not in tag_set:
```

Comment:
Minor note, but what will happen if there would be multiple tags having the same name but different values?

Reply:
It's what we want to have, just remove tags with exactly the same key and same value. And we are designed to support multi-value tags with the same key.